_test/test_transformation/test_ortho_axes.m

```diff
@@ -654,24 +658,31 @@ function test_get_bin_centers_2D_native(~)
             assertEqual(size(centers,2),the_size);
         end
         %
-        function test_worong_char_size_throw(~)
+        function test_worong_mult_throw(~)
             dbr = [-1,-2,-3,0;1,2,3,10];
             bin0 = {[dbr(1,1),0.1,dbr(2,1)];[dbr(1,2),dbr(2,2)];...
                 [dbr(1,3),dbr(2,3)];[dbr(1,4),1,dbr(2,4)]};
             ab = ortho_axes(bin0{:});
-            ex = assertExceptionThrown(@()get_bin_nodes(ab,'-bin_centre',(dbr(2,:)-dbr(1,:))/10),...
+            mult = ones(3,1);
+            mult(mult<1) = 1;
```

Comment:
as mult from L.666 is all ones, this does not seem to do anything. What are you trying to do here?

Reply:
copied from other place. Fixed